programs/cleos/httpc.hpp

```diff
@@ -105,8 +105,10 @@ namespace eosio { namespace client { namespace http {
 
 
    const string history_func_base = "/v1/history";
+   const string trace_api_func_base = "/v1/trace_api";
    const string get_actions_func = history_func_base + "/get_actions";
-   const string get_transaction_func = history_func_base + "/get_transaction";
+   const string get_transaction_func = trace_api_func_base + "/get_transaction";
+   //const string get_transaction_func = history_func_base + "/get_transaction";
```

Comment:
Need to ask prod if cleos should continue to support old `/v1/history/get_transaction` for awhile.